packages/react-resizable-panels-website/playwright.config.ts

```diff
@@ -19,8 +19,10 @@ const config: PlaywrightTestConfig = {
           apiKey: process.env.REPLAY_API_KEY,
           upload: true,
         })
-      : ["line"],
-  ],
+      : undefined,
+    // replicating Playwright's defaults
+    process.env.CI ? (["dot"] as const) : (["list"] as const),
+  ].filter((v): v is NonNullable<typeof v> => !!v),
```

Comment:
with the upcoming TS 5.5 you will be able to rewrite this as:
```suggestion
  ].filter(v => !!v),
```

Reply:
Nice